stdlib/private/SwiftReflectionTest/SwiftReflectionTest.swift

```diff
@@ -211,15 +211,15 @@ internal func sendBytes() {
   let count = Int(readUInt())
   debugLog("Parent requested \(count) bytes from \(address)")
   var totalBytesWritten = 0
-  var pointer = UnsafeMutableRawPointer(bitPattern: address)!
+  var pointer = UnsafeMutableRawPointer(bitPattern: address)
   while totalBytesWritten < count {
     let bytesWritten = Int(fwrite(pointer, 1, Int(count), stdout))
     fflush(stdout)
     if bytesWritten == 0 {
       printErrnoAndExit()
     }
     totalBytesWritten += bytesWritten
-    pointer = pointer.advanced(by: bytesWritten)
+    pointer = pointer?.advanced(by: bytesWritten)
```

Comment:
A better fix might be to check `count` first. It would also be fine to force-unwrap within the loop even if it's not okay to do it at the definition site.